express-server/middleware/errorHandler.js

```diff
@@ -0,0 +1,19 @@
+const { ServerError } = require('../errors/ServerError');
+const { logger } = require('../logger/logger.js');
+const { HTTPStatuses } = require('../constants/HTTPStatus.js');
+
+exports.wrapError = (fn) => (req, res, next) => {
+    Promise.resolve(fn(req, res, next)).catch(next)
+};
+
+exports.errorMiddleware = (err, req, res, next) => {
+  if (err instanceof ServerError) {
+    res.status(err.HTTPStatus).send(err.message);
+    logger.error(`custom server error: ${err.message}`);
```

Comment:
did u check that u saw this message? (after sending the result)

Reply:
yea, i saw this message, but now when i'm looking at it, its weird.

Reply:
i'll change that